edb/lib/sys.edgeql

```diff
@@ -86,6 +90,43 @@ ALTER TYPE sys::Role {
 };
 
 
+CREATE TYPE sys::QueryStats EXTENDING sys::ExternalObject {
+    CREATE LINK branch -> sys::Branch;
+    CREATE PROPERTY query -> std::str;
+    CREATE PROPERTY query_id -> std::int64;
+    CREATE PROPERTY query_type -> sys::QueryType;
+
+    CREATE PROPERTY plans -> std::int64;
+    CREATE PROPERTY total_plan_time -> std::duration;
+    CREATE PROPERTY min_plan_time -> std::duration;
+    CREATE PROPERTY max_plan_time -> std::duration;
+    CREATE PROPERTY mean_plan_time -> std::duration;
+    CREATE PROPERTY stddev_plan_time -> std::duration;
+
+    CREATE PROPERTY calls -> std::int64;
+    CREATE PROPERTY total_exec_time -> std::duration;
+    CREATE PROPERTY min_exec_time -> std::duration;
+    CREATE PROPERTY max_exec_time -> std::duration;
+    CREATE PROPERTY mean_exec_time -> std::duration;
+    CREATE PROPERTY stddev_exec_time -> std::duration;
+
+    CREATE PROPERTY rows -> std::int64;
+    CREATE PROPERTY stats_since -> std::datetime;
```

Comment:
@elprans Elvis, can you add `description` annos to every field? Just one sentence

Reply:
Added in [this crf commit](https://github.com/edgedb/edgedb/pull/7869/commits/eb453cf0772b70cfd894086a8096117e7e305209)